### PR TITLE
Codestyle: move to pycodestyle style checker

### PR DESCRIPTION
**Pull request by bernhardreiter**
http://pep8.readthedocs.io/en/release-1.7.x/ states:

> This package used to be called pep8 but was renamed to pycodestyle to reduce confusion. 

And [pycodestyle](https://pypi.python.org/pypi/pycodestyle) is further maintained and improved since the last pep8 release. I propose to move to pycodestyle with this pull-request.

Because of https://github.com/certtools/intelmq/issues/959 I have added E722 to the warning that are ignored until the issue is resolved.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -12,8 +12,8 @@ all_files  = 1
 [upload_sphinx]
 upload-dir = docs/build/html
 
-[pep8]
-ignore = E221
+[pycodestyle]
```
Comment (wagner-certat) [inline setup.cfg]: At least pycodestyle 2.0.0 still reads from the [pep8] section, for compatibility we could just duplicate this section.

Comment (bernhardreiter) [inline setup.cfg]: I'd rather go fully pycodestyle. Thanks for the review!

Comment (wagner-certat) [inline setup.cfg]: pycodestyle 2.0 (2016-05-31) is also pycodestyle, but it does not recognize the `[pycodestyle]` section in `setup.cfg`. This has been changed in [2.1.0 (2016-11-04)](https://pypi.python.org/pypi/pycodestyle#id4). So for distros not newer than some months, the settings would not apply. I duplicated the section to keep support for these versions. See 59246fbd5f1636797697b1676e08eceb3475ac16

Comment (bernhardreiter) [inline setup.cfg]: For development I believe it is fine to demand an uptodate version, but its your call. Thanks for merging the commit.

**Pull request merged**